src/displayapp/screens/Timer.cpp

```diff
@@ -10,62 +10,32 @@ static void btnEventHandler(lv_obj_t* obj, lv_event_t event) {
   screen->OnButtonEvent(obj, event);
 }
 
-void Timer::CreateButtons() {
-  btnMinutesUp = lv_btn_create(lv_scr_act(), nullptr);
-  btnMinutesUp->user_data = this;
-  lv_obj_set_event_cb(btnMinutesUp, btnEventHandler);
-  lv_obj_set_size(btnMinutesUp, 60, 40);
-  lv_obj_align(btnMinutesUp, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 20, -85);
-  txtMUp = lv_label_create(btnMinutesUp, nullptr);
-  lv_label_set_text_static(txtMUp, "+");
-
-  btnMinutesDown = lv_btn_create(lv_scr_act(), nullptr);
-  btnMinutesDown->user_data = this;
-  lv_obj_set_event_cb(btnMinutesDown, btnEventHandler);
-  lv_obj_set_size(btnMinutesDown, 60, 40);
-  lv_obj_align(btnMinutesDown, lv_scr_act(), LV_ALIGN_IN_LEFT_MID, 20, 35);
-  txtMDown = lv_label_create(btnMinutesDown, nullptr);
-  lv_label_set_text_static(txtMDown, "-");
-
-  btnSecondsUp = lv_btn_create(lv_scr_act(), nullptr);
-  btnSecondsUp->user_data = this;
-  lv_obj_set_event_cb(btnSecondsUp, btnEventHandler);
-  lv_obj_set_size(btnSecondsUp, 60, 40);
-  lv_obj_align(btnSecondsUp, lv_scr_act(), LV_ALIGN_IN_RIGHT_MID, -20, -85);
-  txtSUp = lv_label_create(btnSecondsUp, nullptr);
-  lv_label_set_text_static(txtSUp, "+");
-
-  btnSecondsDown = lv_btn_create(lv_scr_act(), nullptr);
-  btnSecondsDown->user_data = this;
-  lv_obj_set_event_cb(btnSecondsDown, btnEventHandler);
-  lv_obj_set_size(btnSecondsDown, 60, 40);
-  lv_obj_align(btnSecondsDown, lv_scr_act(), LV_ALIGN_IN_RIGHT_MID, -20, 35);
-  txtSDown = lv_label_create(btnSecondsDown, nullptr);
-  lv_label_set_text_static(txtSDown, "-");
-}
-
-Timer::Timer(DisplayApp* app, Controllers::TimerController& timerController)
-  : Screen(app), running {true}, timerController {timerController} {
+Timer::Timer(DisplayApp* app, Controllers::TimerController& timerController) : Screen(app), timerController {timerController} {
 
   time = lv_label_create(lv_scr_act(), nullptr);
   lv_obj_set_style_local_text_font(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &jetbrains_mono_76);
-  lv_obj_set_style_local_text_color(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_MAKE(0xb0, 0xb0, 0xb0));
+  lv_obj_set_style_local_text_color(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_WHITE);
+  lv_label_set_text_static(time, "00:00");
```

Comment:
we could replace the text by adding a function `Counter::HideControls()` and `Counter::ShowControls()` to not hide the whole counter-object, but tell the counter it should just hide the controls. Then we don't need the `time` object anymore